build/yamls/elastiflow/logstash/logstash.conf

```diff
@@ -0,0 +1,1595 @@
+              event.set( '[flow][bytes]', event.get('[ipfix][initiatorOctets]').to_i + event.get('[ipfix][responderOctets]').to_i )
+            "
+          }
+        } else {
+          mutate {
+            id => "ipfix_add_initiatorOctets"
+            replace => { "[flow][bytes]" => "%{[ipfix][initiatorOctets]}" }
+          }
+        }
+      } else if [ipfix][responderOctets] {
+        mutate {
+          id => "ipfix_add_responderOctets"
+          replace => { "[flow][bytes]" => "%{[ipfix][responderOctets]}" }
+        }
+      }
+      if [flow][bytes] {
+        mutate {
+          id => "ipfix_convert_bytes"
+          convert => { "[flow][bytes]" => "integer" }
+        }
+      }
+
+    # Populate normalized ElastiFlow fields with packets if available.
+      if [ipfix][packetDeltaCount] {
+        if [ipfix][reversePacketDeltaCount] {
+          ruby {
+            id => "ipfix_normalize_packets_from_packetDeltaCount_reversePacketDeltaCount"
+            code => "
+              event.set( '[flow][packets]', event.get('[ipfix][packetDeltaCount]').to_i + event.get('[ipfix][reversePacketDeltaCount]').to_i )
+            "
+          }
+        } else {
+          mutate {
+            id => "ipfix_add_packetDeltaCount"
+            replace => { "[flow][packets]" => "%{[ipfix][packetDeltaCount]}" }
+          }
+        }
+      } else if [ipfix][reversePacketDeltaCount] {
+        mutate {
+          id => "ipfix_add_reversePacketDeltaCount"
+          replace => { "[flow][packets]" => "%{[ipfix][reversePacketDeltaCount]}" }
+        }
+      } else if [ipfix][initiatorPackets] {
+        if [ipfix][responderPackets] {
+          ruby {
+            id => "ipfix_normalize_packets_from_initiatorPackets_responderPackets"
+            code => "
+              event.set( '[flow][packets]', event.get('[ipfix][initiatorPackets]').to_i + event.get('[ipfix][responderPackets]').to_i )
+            "
+          }
+        } else {
+          mutate {
+            id => "ipfix_add_initiatorPackets"
+            replace => { "[flow][packets]" => "%{[ipfix][initiatorPackets]}" }
+          }
+        }
+      } else if [ipfix][responderPackets] {
+        mutate {
+          id => "ipfix_add_responderPackets"
+          replace => { "[flow][packets]" => "%{[ipfix][responderPackets]}" }
+        }
+      }
+
+    # If sampled IPFIX, adjust Bytes and Packets accordingly.
+      # Workaround for sampled flows when the sampling interval is not set
+      if ![flow][sampling_interval] {
+        translate {
+          dictionary_path => "${ELASTIFLOW_DICT_PATH:/etc/logstash/elastiflow/dictionaries}/sampling_interval.yml"
+          field => "[node][ipaddr]"
+          destination => "[flow][sampling_interval]"
+          fallback => "0"
+          refresh_behaviour => "replace"
+        }
+        mutate {
+          convert => { "[flow][sampling_interval]" => "integer" }
+        }
+      }
+
+      if [flow][sampling_interval] {
```

Comment:
We are not supporting ipfix sampling for now. Do we still need the related code in conf? Will this increase unnecessary overhead in elastiflow?

Reply:
Thanks for the info, I will remove these part in that case.